### ISSUE TITLE
bug: conflict with pocco81/auto-save.nvim yank triggers write

### ISSUE THREAD

**Issue by mikesmithgh** [mikesmithgh/kitty-scrollback.nvim#196]
Result of discussion: https://github.com/mikesmithgh/kitty-scrollback.nvim/discussions/192

lazy.nvim setup:
```lua
return {
  'pocco81/auto-save.nvim',
  lazy = false,
  enabled = true,
  config = function()
    require('auto-save').setup()
  end,
}
```

### Steps to reproduce:
- open kitty-scrollback.nvim
- yank text

### Actual:
- text is copied to "paste window"
- auto-save.nvim triggers a write to try and autosave
- write results in sending the contents of the "paste window" to kitty

### Expected:
- text is copied to "paste window"

### Workaround:
Disable [pocco81/auto-save.nvim](https://github.com/Pocco81/auto-save.nvim)

**Comment by mikesmithgh** [comment]
I created a Wiki page [Recommended Configurations](https://github.com/mikesmithgh/kitty-scrollback.nvim/wiki/Recommended-Configurations) provide recommendations for plugins that may conflict with kitty-scrollback.nvim.

The recommend configuration for auto-save.nvim is:

Disable auto-save.nvim when kitty-scrollback.nvim is active.

```lua
require('auto-save').setup({
  enabled = vim.env.KITTY_SCROLLBACK_NVIM ~= 'true',
})
```

This lets you keep using auto-save.nvim in normal nvim sessions but disables it for kitty-scrollback.nvim sessions.

FYI @aksh1618

**Comment by aksh1618** [comment]
> Disable auto-save.nvim when kitty-scrollback.nvim is active.

@mikesmithgh I had tried exactly that as my first step! But sadly it didn't work, so I decided to remove the plugin for now and file it under see later. It is now later, and I found out that this plugin doesn't read the config correctly (See https://github.com/pocco81/auto-save.nvim/issues/76) and there is another fork that is more maintained: https://github.com/okuuva/auto-save.nvim. So I switched to that one and it works!
